src/Linux/debian/changelog

```diff
@@ -1,7 +1,7 @@
 az-dcap-client (1.7) stable; urgency=medium
```

Comment:
Please make a new version instead of copying over previous if you are going to make a new release

Reply:
We did not release 1.7.0 before. So I changed the description and make a release of 1.7.0 version

Reply:
Thank you for clarifying :) I'll approve